bears/ruby/RuboCopBear.py

```diff
@@ -28,8 +28,188 @@ def create_arguments(filename, file, config_file, rubocop_config: str=""):
+                        ignore_unused_block_args_if_empty: bool=True,
+                        allow_unused_block_keyword_arguments: bool=False,
+                        ignore_unused_method_args_if_empty: bool=True,
+                        allow_unused_method_keyword_args: bool=False):
+        """
+        Not all settings added.
+        Notable settings missing: Rails settings.
+
+        :param access_modifier_indentation:
+            Indent private/protected/public as deep as method definitions
+            options:
+                ``indent`` :  Indent modifiers like class members.
+                ``outdent`` : Indent modifiers one level less than
+                              class members.
+        :param preferred_alias:
+            Which method to use for aliasing in ruby?
+            options : ``alias`` , ``alias_method``.
+        :param align_hash_rocket_by:
+            Alignment of entries using hash rocket as separator.
+        :param align_colon_by:
+            Alignment of entries using colon as separator.
+        :param inspect_last_argument_hash:
+            Select whether hashes that are the last argument in a method call
+            should be inspected?
+            options: 'always_inspect', 'always_ignore', 'ignore_implicit',
+                     'ignore_explicit'.
+        :param align_parameters:
+            Alignment of parameters in multi-line method calls.
+
+            The `with_first_parameter` style aligns the following lines along
+            the same column as the first parameter.
+
+            The `with_fixed_indentation` style aligns the following lines
+            with one level of indentation relative to the start of the line
+            with the method call.
+        :param class_check:
+            How to check type of class.
+            options: 'is_a?', 'kind_of?'.
+        :param comment_keywords:
+            Checks formatting of special comments based on keywords like
+            TODO, FIXME etc.
+        :param min_if_unless_guard:
+            The number of lines that are tolerable within an if/unless block,
+            more than these lines call for the usage of a guard clause.
+        :param indentation_width:
+            No. of spaces to indent.
+        :param method_name_case:
+            Case of a method's name.
+            options: 'snake_case', 'camelCase'.
+        :param string_literals:
+            Use ' or " as string literals.
+            options: 'single_quotes', 'double_quotes'.
+        :param variable_name_case:
+            Case of a variable's name.
+            options: 'snake_case', 'camelCase'.
+        :param max_class_length:
+            Max lines in a class.
+        :param class_length_count_comments:
+            Count comments while calculating class length?
+        :param max_module_length:
+            Max lines in a module.
+        :param module_length_count_comments:
+            Count comments while calculating module length?
```

Comment:
Similar thing with the question mark. I get that you want to tell me that this is a bool value, but it really looks like you were unsure about the option :smile: 

I'd go with:
"Whether or not to count comments..." or something similar.

Again I would consider this fine if it is worded like this in Rubocop.